src/Symfony/Bundle/FrameworkBundle/Command/ContainerLintCommand.php

```diff
@@ -64,7 +65,15 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $container->setParameter('container.build_time', time());
 
-        $container->compile();
+        try {
+            $container->compile();
+        } catch (InvalidArgumentException $e) {
+            $errorIo->error($e->getMessage());
+
+            return 1;
+        }
+
+        $errorIo->success('The container was lint successfully: all services are injected with values that are compatible with their type declarations.');
```

Comment:
shouldnt it be the regular $io?

Reply:
You are right and we did that mistake in multiple places, I'm gonna fix them.